fe/fe-core/src/main/java/com/starrocks/statistic/StatisticExecutor.java

```diff
@@ -208,9 +208,13 @@ public AnalyzeStatus collectStatistics(StatisticsCollectJob statsJob) {
                 db.getId(), table.getId(), columns,
                 statsJob.getType(), statsJob.getScheduleType(), statsJob.getProperties(),
                 LocalDateTime.now());
-        analyzeStatus.setStatus(StatsConstants.ScheduleStatus.RUNNING);
+        analyzeStatus.setStatus(StatsConstants.ScheduleStatus.FAILED);
```

Comment:
In order to prevent the fe restart interruption of the process, we first set the task state to FAIL. If the process is not interrupted, there must be an end state setting, such as SUCCESS or FAIL. The RUNNING in the next line only modifies the state in memory and does not record the log. In this way, after the FE restarts, the unfinished statistics collection task will be FAIL instead of RUNNING.